mtest/README.md

```diff
@@ -6,7 +6,7 @@ To build all tests:
 
 | Linux | OSX | Windows (MinGW) |
 | ----- | --- | --------------- |
-| make debug<br>sudo make installdebug<br>cd build.debug/mtest<br>make</pre> | make -f Makefile.osx debug<br>make -f Makefile.osx installdebug<br>cd build.debug/mtest<br>make -f Makefile.osx | mingw32-make -f Makefile.mingw debug<br>mingw32-make -f Makefile.mingw installdebug<br>cd build.debug\mtest<br>mingw32-make -f Makefile.mingw |
+| make debug<br>sudo make installdebug<br>cd build.debug/mtest<br>make</pre> | make -f Makefile.osx debug<br>make -f Makefile.osx installdebug<br>cd build.debug/mtest<br>make -f Makefile.osx | mingw32-make -f Makefile.mingw debug<br>mingw32-make -f Makefile.mingw installdebug<br>cd build.debug\mtest<br>mingw32-make -f Makefile |
```

Comment:
Wouln't that break the MINGW builds?

Reply:
The top level `Makefile.mingw` still needs to be invoked and that one works. However, in the resulting build directory, under the mtest folder there is no `Makefile.mingw` generated, the file is simply called `Makefile`.